picard/ui/widgets/scriptdocumentation.py

```diff
@@ -138,7 +138,7 @@ def process_tag(tag: TagVar):
             return f'<dt>{tag_title}</dt><dd>{tag_desc}</dd>'
 
         html = ''
-        for tag in sorted(ALL_TAGS, key=lambda x: x.script_name()):
+        for tag in sorted(ALL_TAGS, key=lambda x: x.script_name().strip('_')):
```

Comment:
You should use lstrip()

Reply:
You're right.  I was looking for that but kept trying stripl() instead.  Duh.